app/scripts/directives/header/view.html

```diff
@@ -2,6 +2,12 @@
 <md-content>
   <link rel="stylesheet" href="app/scripts/directives/header/style.css">
 
+  <md-toolbar class="md-hue-2" md-colors="::{backgroundColor: 'default-primary-900'}">
+    <div flex="100" layout="row" layout-align="center center">
+      <span flex layout="row" layout-align="center center" md-truncate class="md-display-1">Immunisation App</span>
```

Comment:
Lets update this line with
`<span layout="row" layout-align="center center" md-truncate class="md-display-1"><md-icon>local_hospital</md-icon>OpenCRVS Immunisation</span>`